static/css/shared/modal.scss

```diff
@@ -20,16 +20,20 @@
 @import "node_modules/bootstrap/scss/variables";
 @import "node_modules/bootstrap/scss/mixins";
 
-$border: 0.5px solid #dee2e6;
+$border: 1px solid #dadce0;
 
 .modal textarea {
-  background: #efefef;
-  border-radius: 3px;
+  background: #fff;
+  border-radius: 8px;
   border: $border;
   display: flex;
-  font-family: monospace;
+  font-family: "Google Sans Text", sans-serif;
```

Comment:
Use css variable here - might need to verify which one, but maybe `font-family: var(--dc-font-family, base.$font-family-sans-serif);`